src/junitParser.ts

```diff
@@ -70,12 +70,11 @@ async function parseFile(file: string, projectTokenDictionaryStrs: string[]) {
     const data: string = fs.readFileSync(file, 'utf8');
     const parser = new XMLParser({ allowBooleanAttributes: true, ignoreAttributes: false, attributeNamePrefix: '' });
     const report = parser.parse(data) as Partial<JUnitReport>;
-
-    return parseSuite(report, file, projectTokenDictionaryStrs);
+    const testsuites = castArray(report?.testsuite || report.testsuites?.testsuite);
```

Comment:
Still no reason for optional chaining AFAICT?
```suggestion
    const testsuites = castArray(report.testsuite || report.testsuites?.testsuite);
```